mobile/src/calendar/styles.ts

```diff
@@ -83,18 +83,39 @@ export const calendarStyles = StyleSheet.create({
     },
     weekDayText: {
         fontSize: 12,
-        lineHeight: 14
+        lineHeight: 14,
+        color: 'rgba(0, 0, 0, 0.5433)'
     }
 });
 
-export const calendarIntervalColors = {
-    vacation: '#2F80ED',
-    sickLeave: '#F2C94C',
-    dayoff: '#EB5757'
-};
+export class CalendarIntervalColor {
+    public static vacation = '#2F80ED';
+    public static sickLeave = '#F2C94C';
+    public static dayoff = '#EB5757';
+    public static additionalWork = '#18515E'; //TODO: replace with the right color
+
+    public static getColor(type: CalendarEventsType) {
+        switch (type) {
+            case CalendarEventsType.Vacation:
+                return CalendarIntervalColor.vacation;
+
+            case CalendarEventsType.SickLeave:
+                return CalendarIntervalColor.sickLeave;
+
+            case CalendarEventsType.Dayoff:
+                return CalendarIntervalColor.dayoff;
+
+            case CalendarEventsType.AdditionalWork:
+                return CalendarIntervalColor.additionalWork;
+
+            default:
+                return null;
+        }
+    }
+}
 
 // serjKim: Depends on count of overlapped intervals?..
-const intervalOpacity = 0.5;
+const intervalOpacity = .95;
```

Comment:
?

Reply:
When opacity is .5 the colors are too pale and do not match the design at all

Reply:
But it's difficult to determine when some overlapped interval is started or finished